charts/argo-rollouts/templates/controller/role.yaml

```diff
@@ -150,4 +150,15 @@ rules:
   - get
   - update
   - patch
+# traefikservices needed for using trafficRouting with traefik
+- apiGroups:
+  - traefik.containo.us
+  resources:
+  - traefikservices
+  verbs:
+  - list
+  - watch
+  - get
+  - update
+  - patch
```

Comment:
Can you test with the upstream verbs only (watch, get, update)?

https://github.com/argoproj/argo-rollouts/blob/v1.3.2/manifests/role/argo-rollouts-clusterrole.yaml#L224-L231